src/status_im/ui/screens/offline_messaging_settings/edit_mailserver/views.cljs

```diff
@@ -60,14 +62,21 @@
             :default-value   name
             :on-change-text  #(re-frame/dispatch [:mailserver.ui/input-changed :name %])
             :auto-focus      true}]
-          [text-input/text-input-with-label
-           {:label           (i18n/label :t/mailserver-address)
-            :placeholder     (i18n/label :t/specify-mailserver-address)
-            :content         qr-code
-            :style           styles/input
-            :container       styles/input-container
-            :default-value   url
-            :on-change-text  #(re-frame/dispatch [:mailserver.ui/input-changed :url %])}]
+          [react/view
+           {:flex 1}
+           [text-input/text-input-with-label
+            {:label          (i18n/label :t/mailserver-address)
+             :placeholder    (i18n/label :t/specify-mailserver-address)
+             :content        qr-code
+             :style          styles/input
+             :container      styles/input-container
+             :default-value  url
+             :on-change-text #(re-frame/dispatch [:mailserver.ui/input-changed :url %])}]
+           (when (and (not (string/blank? url)) (not is-valid?))
+             [tooltip/tooltip (i18n/label :t/invalid-format)
```

Comment:
enode and mailserver don't have the same format, mailserver wants a password:
`enode://{enode-id}:{password}@{ip-address}:{port}` 
so we might want different messages

Reply:
sure, thanks

Reply:
so for bootnode it is `enode://{enode-id}@{ip-address}:{port}`, correct?

Reply:
yes, I believe so

Reply:
ok, fixed